Automattic/jetpack
# VideoPress: re-write token bridge lib

This PR re-writes the token bridge library in order to know its functionality:
* Use TypeScript
* Add debug() lines

Fixes #

## Proposed changes:

* VideoPress: re-write token bridge lib

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Open the network tab
* filter by `token-bridge`
* Confirm the app loads the script a few times. We need to polish that, btw. Follow-up.
* Open the console and filter by `token-bridge` too
* Hard refresh
* Add/edit a video block
* Change the video privacy
* Check how the bridge listens and responds when the client requests the token
* Check with `v5` and `v6`. In the block editor context and the front-end.

https://user-images.githubusercontent.com/77539/215863982-6fae4190-509c-4535-bb35-41bd7ea6fb08.mov
